sygnal/webpushpushkin.py

```diff
@@ -137,6 +138,11 @@ async def _dispatch_notification_unlimited(self, n, device, context):
         payload = WebpushPushkin._build_payload(n, device)
         data = json.dumps(payload)
 
+        # note that webpush modifies vapid_claims, so make sure it's only used once
+        vapid_claims = {
+            "sub": "mailto:{}".format(self.vapid_contact_email),
+            "exp": self._get_vapid_exp(),
```

Comment:
This seems fine, but I do wonder if we should leave manually specifying `exp` out until we get the tests going as it isn't necessary to copy this out of pywebpush yet.